pkg/informers/caicloud/interface.go

```diff
@@ -0,0 +1,26 @@
+package caicloud
```

Comment:
Why `caicloud`?  This is not a good package name for OSS (as well as company name in between the path).  Think of another name?  I know this is *very* difficult :)